sentry-android-core/src/main/java/io/sentry/android/core/AndroidOptionsInitializer.java

```diff
@@ -14,7 +14,7 @@ static void init(SentryOptions options, Context context, ILogger logger) {
     options.setLogger(logger);
 
     // TODO this needs to fetch the data from somewhere - defined at build time?
-    options.setSentryClientName("sentry-android/0.0.1");
+    options.setSentryClientName("sentry.java.android/0.0.1");
```

Comment:
why is it `sentry.java.android`?

Reply:
We name our sdks in the format `sentry.tech` like:
`sentry.dotnet`
`sentry.php`

When we publish packages that are specialization of it, it takes the format:
`sentry.dotnet.aspnetcore`
`sentry.javascript.browser`

Since users of the pure `sentry.java` SDK will have that as an SDK name, to help us easily differentiate what was installed in the app, for android @HazAT proposed `sentry.java.android`.

If it was rewritten in kotlin one day, it'd change to `sentry.kotlin.android` :)

Reply:
what's about the version? should we also use `BuildConfig.VERSION_NAME`?

Reply:
Yes, I'll merge this before we discuss that thouhg